### ISSUE TITLE
Can't add read-only locations

### ISSUE THREAD

**Issue by TwanLuttik** [spacedriveapp/spacedrive#1738]
### Describe the bug

I was trying to add a location from a hard drive that is formatted for windows _(forgot what the format is called)_ and is just read only so it can't write the `.spacedrive` file

![CleanShot 2023-11-04 at 12 28 40@2x](https://github.com/spacedriveapp/spacedrive/assets/19343894/b3f5bbb5-1644-4ad1-913b-14fba48ad535)

### Reproduction

- Add a read only location

### Platform and versions

```Shell
8.10.2
cargo 1.72.1
rustc 1.72.1 (d5c2e9c34 2023-09-13) (Homebrew)
```

**Comment by utkubakir** [comment]
You can go to Privacy > Full Disk Access to make it work for now. it's tracked here https://github.com/spacedriveapp/spacedrive/issues/1740

**Comment by TwanLuttik** [comment]
Maybe I have worded this wrong but its a HDD formatted as NTFS so mac can't write to it.

**Comment by utkubakir** [comment]
> Maybe I have worded this wrong but its a HDD formatted as NTFS so mac can't write to it.

Right... This will be discussed internally, it would be nice to just get rid of `.spacedrive` files imo

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
